packages/template-retail-react-app/scripts/translations/utils.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint @typescript-eslint/no-var-requires: "off" */
+const fs = require('fs')
+const path = require('path')
+
+const getOutputFolder = () => {
+    const packagePath = path.join(process.cwd(), 'package.json')
+    const pkgJSON = JSON.parse(fs.readFileSync(packagePath))
+
+    const overridesDir = pkgJSON.ccExtensibility?.overridesDir
+    const outputFolder = overridesDir
+        ? path.join(overridesDir, 'app/static/translations/compiled')
+        : 'app/static/translations/compiled'
```

Comment:
We always check that **both** the `overridesDir` **and** `extends` keys are set, because the feature doesn't work without both
```suggestion
    const overridesDir = pkgJSON.ccExtensibility?.overridesDir
    const outputFolder = pkgJSON?.ccExtensibility?.overridesDir && pkgJSON?.ccExtensibility?.extends
        ? path.join(overridesDir, 'app/static/translations/compiled')
        : 'app/static/translations/compiled'
```